packages/common/src/converter/payload-converters.ts

```diff
@@ -87,3 +141,27 @@ export class BinaryPayloadConverter implements PayloadConverterWithEncoding {
     return content.data as any;
   }
 }
+
+export const searchAttributePayloadConverter = new WrappedPayloadConverter(new JsonPayloadConverter());
+
+export class DefaultPayloadConverter extends CompositePayloadConverter {
+  // Match the order used in other SDKs, but exclude Protobuf converters so that the code, including
+  // `proto3-json-serializer`, doesn't take space in Workflow bundles that don't use Protobufs. To use Protobufs, use
+  // {@link DefaultPayloadConverterWithProtobufs}.
+  //
+  // Go SDK:
+  // https://github.com/temporalio/sdk-go/blob/5e5645f0c550dcf717c095ae32c76a7087d2e985/converter/default_data_converter.go#L28
+  constructor() {
+    super(new UndefinedPayloadConverter(), new BinaryPayloadConverter(), new JsonPayloadConverter());
+  }
+}
+
+/**
+ * The default {@link PayloadConverter} used by the SDK.
+ * Supports `Uint8Array` and JSON serializables (so if [`JSON.stringify(yourArgOrRetval)`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify#description) works, the default payload converter will work).
+ *
+ * To also support Protobufs, create a custom payload converter with {@link DefaultPayloadConverter}:
+ *
+ * `const myConverter = new DefaultPayloadConverter({ protobufRoot })`
+ */
+export const defaultPayloadConverter = new WrappedPayloadConverter(new DefaultPayloadConverter());
```

Comment:
Let's not wrap the default, instead I'd wrap in the worker / workflow runtime / client, that way any converter we accept would be wrapped.

Reply:
Like this?

https://github.com/temporalio/sdk-typescript/pull/558/commits/a59fe705c366852c043ffb4dcba6338c2847187a